### PR TITLE
added back4app API for BLS job search

### PR DESCRIPTION
**Pull request by carsonwolber**
Screenshot 2024-06-30 at 10 34 41 PM

### CODE REVIEW

File: frontend/src/ProfilePage/FormPages/PageOne.jsx [hunk 1]
```diff
@@ -1,119 +1,148 @@
...
+                "title": {
+                    "$regex": searchTerm,
+                    "$options": "i"
+                }
+            }));
+            const headers = {
+                'X-Parse-Application-Id': import.meta.env.VITE_API_BLS_ID,
+                'X-Parse-REST-API-Key': import.meta.env.VITE_API_BLS_KEY
+            };
+            fetch(`https://parseapi.back4app.com/classes/Occupations_Job?limit=10&keys=title&where=${where}`, {
```
Comment (pangers) [inline frontend/src/ProfilePage/FormPages/PageOne.jsx:40]: Would it be worth pulling this API call out into a separate file, like you have been doing with the other API calls?

**Review by pangers — COMMENTED** [review]
One habit to get into is to add a description when making a PR to explain any thought process or and to explain what the changes here do. Many times people reviewing your code won't have context on what you are doing so explaining it in a description will help them have a baseline understanding of what is going on. Could you explain what a BLS job search is? Could you explain what back4app API does?

**Comment by William-La** [comment]
great callout by James-- going forward, let's add a quick summary or context comment in addition to your test plan comment on PRs that are more technical.

File: frontend/src/ProfilePage/FormPages/PageOne.jsx (hunk 1 above)
Comment (carsonwolber) [reply frontend/src/ProfilePage/FormPages/PageOne.jsx:40]: Yes, good point, I'll clean that up now

**Pull request merged**